spec/intl.html

```diff
@@ -2139,13 +2182,22 @@ <h1>Temporal.Calendar.prototype.mergeFields ( _fields_, _additionalFields_ )</h1
             1. Perform ? RequireInternalSlot(_calendar_, [[InitializedTemporalCalendar]]).
             1. Set _fields_ to ? ToObject(_fields_).
             1. Set _additionalFields_ to ? ToObject(_additionalFields_).
-            1. If _calendar_.[[Identifier]] is *"iso8601"*, then
-              1. Return ? DefaultMergeCalendarFields(_fields_, _additionalFields_).
-            1. Let _fieldsCopy_ be OrdinaryObjectCreate(%Object.prototype%).
-            1. Perform ? CopyDataProperties(_fieldsCopy_, _fields_, « », « *undefined* »).
-            1. Let _additionalFieldsCopy_ be OrdinaryObjectCreate(%Object.prototype%).
+            1. Let _additionalFieldsCopy_ be OrdinaryObjectCreate(*null*).
             1. Perform ? CopyDataProperties(_additionalFieldsCopy_, _additionalFields_, « », « *undefined* »).
-            1. Return CalendarDateMergeFields(_calendar_.[[Identifier]], _fieldsCopy_, _additionalFieldsCopy_).
+            1. NOTE: Every property of _additionalFieldsCopy_ is a data property with non-*undefined* value, but some property keys may be Symbols.
+            1. Let _excludedKeys_ be « ».
+            1. Let _additionalKeys_ be ! _additionalFieldsCopy_.[[OwnPropertyKeys]]().
+            1. For each element _key_ of _additionalKeys_, do
+              1. Append _key_ to _excludedKeys_.
+              1. If _calendar_.[[Identifier]] is *"iso8601"*, then
+                1. If _key_ is *"month"*, append *"monthCode"* to _excludedKeys_.
+                1. Else if _key_ is *"monthCode"*, append *"month"* to _excludedKeys_.
+              1. Else,
+                1. Set _excludedKeys_ to the list-concatenation of _excludedKeys_ and CalendarMutuallyExclusiveKeys(_calendar_, _key_).
```

Comment:
Is it OK that `CalendarMutuallyExclusiveKeys` can return the same property multiple times? For example, if `_additionalFields_` includes both `era` and `eraYear`, then `year` will be added twice to `_excludedKeys_`.  And for Japanese, if `_additionalFields_` includes both `day` and `year` then I think `era` and `eraYear` will be present 3x in `_excludedKeys_`.

Reply:
I guess there's a [Set specification type](https://tc39.es/ecma262/#sec-set-and-relation-specification-type) we can use for this.